### DISCUSSION TITLE
Setup Booting After Every Login Bug

### DISCUSSION THREAD

**Discussion by EddieN17** [Security-Onion-Solutions/securityonion#10639] [2.4]
I'm using the SO ISO, running 2.4.2. I have a distributed setup going running on VMWare ESXi VMs. Whenever I ssh into the search or forward nodes using the setup administrative account, it always tries booting up into setup, even though the nodes have already successfully gone through setup. This issue doesn't exist when I log into the manager node. The search and forward machines have been running for awhile, gone through restarts, but issue still exists.

My fix: Manually remove the startup command in the .bash_profile as it still existed in there.

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/75918131/a8bfc7c1-6b7a-40fe-84a7-70ee77823476)

**Comment by dougburks** [comment]
This should be resolved in the next release:
https://github.com/Security-Onion-Solutions/securityonion/issues/10498
